### PR TITLE
docs(321): Update SD-In-A-Box image

### PR DESCRIPTION
**Pull request by Filbird**
Small edit to clarify instructions for running SD-In-A-Box

### CODE REVIEW

File: docs/cluster-management/running-locally.md [hunk 1]
```diff
@@ -20,7 +20,13 @@ This handy feature will bring up an entire Screwdriver instance (UI, API, and lo
 
 1. Follow instructions in the [SD-in-a-box docs](https://github.com/screwdriver-cd/in-a-box#screwdriver-in-a-box).
 
-2. You will be prompted to enter your Client ID and Client Secret. Afterwards, type `y` to launch Screwdriver!
+2. You will be prompted to enter your desired SCM provider as well as your Client ID and Client Secret. Afterwards, type `y` to launch Screwdriver!
```
Comment (tkyi) [inline docs/cluster-management/running-locally.md]: Would this make more sense to put in https://github.com/screwdriver-cd/in-a-box/blob/master/README.md?

**Review by pranavrc — APPROVED** [review]
image

File: docs/cluster-management/running-locally.md (hunk 1 above)
Comment (Filbird) [reply docs/cluster-management/running-locally.md]: Perhaps! Would you prefer if I moved it there? :)

Comment (tkyi) [reply docs/cluster-management/running-locally.md]: yes please!

Comment (Filbird) [reply docs/cluster-management/running-locally.md]: https://github.com/screwdriver-cd/in-a-box/pull/2

**Review by minzcmu — APPROVED** [review]
👍

**Pull request merged**